composer.json

```diff
@@ -20,7 +20,8 @@
   "require": {
     "php": ">=7.3",
     "ext-json": "*",
-    "elegant-bro/interfaces": "^1.0"
+    "elegant-bro/interfaces": "^1.0",
+    "elegant-bro/stringify": "^0.3.1"
```

Comment:
we have to avoid circular dependencies between libs

Reply:
Done!